src/Action/BaseAction.php

```diff
@@ -26,6 +26,15 @@
     protected $_responding = false;
 
     /**
+     * Default configuration
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'modelFactory' => 'Table',
```

Comment:
:+1: for @ADmad 's suggestion 

Also remember to update the docs with the new option :)

I know @lorenzo also is working on getting crud / crud-view to play nice with ElasticSearch :)